tests/src/Interop/PInvoke/ExactSpelling/CMakeLists.txt

```diff
@@ -1,8 +1,13 @@
-cmake_minimum_required (VERSION 2.6) 
-project (ExactSpellingNative) 
-include ("${CLR_INTEROP_TEST_ROOT}/Interop.cmake") 
```

Comment:
I would prefer if you could keep this line. See the `readme.md` at the root of the Interop test dir for details.